specification/compute/resource-manager/Microsoft.Compute/stable/2022-01-03/GalleryRP/gallery.json

```diff
@@ -2926,6 +2886,23 @@
         "operationType"
       ],
       "x-ms-azure-resource": true
+    },
+    "GalleryProvisioningState": {
```

Comment:
should this also be tagged as ReadOnly? 
If it is intended to never be edited in the future in any possible case, you can tag it as readonly. Going from ReadOnly to not ReadOnly is easier than the reverse.

Reply:
added, and updated SDK pr, there is no update for this field.

Reply:
Good catch, thanks @Sandido